Documentation/observability/metrics.rst

```diff
@@ -526,10 +526,14 @@ Context Options
 ^^^^^^^^^^^^^^^
 
 Hubble metrics support configuration via context options.
-Currently there are 3 supported context options for all metrics:
+Supported context options for all metrics:
 
 - ``sourceContext`` - Configures the ``source`` label on metrics.
```

Comment:
Might be helpful to clarify a bit here by adding `for both egress and ingress traffic`. Same with `destinationContext` [1].
```suggestion
- ``sourceContext`` - Configures the ``source`` label on metrics for both egress and ingress traffic.
```

Reply:
done